numcompress/numcompress.py

```diff
@@ -1,5 +1,8 @@
-PRECISION_LOWER_LIMIT=0
+import numpy as np
+
+PRECISION_LOWER_LIMIT = 0
 PRECISION_UPPER_LIMIT = 10
+SEPARATOR = ','  # This char should have Unicode code point in the range [32, 63) to avoid overlaps.
```

Comment:
Nitpick for the comments:

```
# Used for N-Dimensional array. Separates dimension and the series.
# Should have ASCII value between (0, 63) to avoid overlapping with regular compress output.
SEPARATOR = ','  
```